core/utilities/mcmlln-tools.rb

```diff
@@ -52,7 +53,7 @@ def self.makeDir(directory)
 
     # An array listing everything in a directory
     def self.dirList(directory)
-      Dir.entries(directory)
+      Dir.entries(directory) - ['..', '.']
```

Comment:
It still lists all entries in a Dir, but excludes the current directory '.' and parent directory '..'
I think having them as members of the array was throwing off the count of an array somewheres, which I notices while adding logging.  I will add a comment! :)